OtripleS.Portal.Web/Services/StudentViews/IStudentViewService.cs

```diff
@@ -3,8 +3,8 @@
 // FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
 // ---------------------------------------------------------------
 
-using System.Threading.Tasks;
 using OtripleS.Portal.Web.Models.StudentViews;
+using System.Threading.Tasks;
```

Comment:
You need to enable a setting to ensure 'System' directives are already on the top.

Reference: https://dailydotnettips.com/automatically-place-the-system-directives-first-when-sorting-usings-option-in-visual-studio/

Reply:
Thanks for the suggestion. Fixed it